config/formats.js

```diff
@@ -206,6 +206,28 @@ let Formats = [
 			'Arceus', 'Reshiram', 'Zekrom', 'Kyurem', 'Xerneas', 'Yveltal', 'Solgaleo', 'Lunala', 'Necrozma-Dusk-Mane', 'Necrozma-Dawn-Wings',
 		],
 	},
+	{
+		name: "[Gen 7] Let's Go! Cup",
+		threads: [`&bullet; <a href="https://www.smogon.com/forums/threads/3639342/">Let&apos;s Go! Cup</a>`],
+
+		mod: 'gen7',
+		forcedLevel: 50,
+		teamLength: {
+			validate: [1, 1],
+			battle: 1,
+		},
+		ruleset: ['Pokemon', 'Cancel Mod'],
+		banlist: ['Illegal', 'Unreleased'],
+		onValidateTeam: function (team) {
```

Comment:
This can just be `onValidateSet: function (set, format)`

Reply:
oh, I see why. There can only be one Pokemon per team. You should just have a check that says ``if (team.length !== 1) return [`Only one Pokemon per team is allowed.`];``